pkg/hooks/connection/tracker.go

```diff
@@ -213,19 +209,32 @@ func (conn *Tracker) IsComplete() (bool, []byte, []byte, time.Time, time.Time) {
 	}
 
 	var reqTimestampTest time.Time
-	// Checking if record traffic is recorded and reqeust timestamp is captured or not.
-	if recordTraffic && len(conn.reqTimestampTest) > 0 {
-		// Get the timestamp of current request
-		reqTimestampTest = conn.reqTimestampTest[0]
-		// Pop the timestamp of current request
-		conn.reqTimestampTest = conn.reqTimestampTest[1:]
+	var resTimestampTest time.Time
+	// Checking if record traffic is recorded and request & response timestamp is captured or not.
+	if recordTraffic {
+		if len(conn.reqTimestampTest) > 0 {
+			// Get the timestamp of current request
+			reqTimestampTest = conn.reqTimestampTest[0]
+			// Pop the timestamp of current request
+			conn.reqTimestampTest = conn.reqTimestampTest[1:]
+		} else {
+			conn.logger.Debug("no request timestamp found, skipping recording")
```

Comment:
Please add some context about the request that are skipping. This will the user understand which request we skipped. Maybe url, method. 

The message can also be something like, "failed to capture request timestamp for a request. Please record it again if important"

Reply:
Also, please change this to info rather than debug. The user should know about it.

Reply:
Sure, on it